<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/img/منابع پروژه/to-do.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string>     Remind  Everythings!!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>260</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pass">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>      Do you not have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
font: 12pt &quot;Dast Nevis&quot;;</string>
    </property>
    <property name="text">
     <string>signup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eror">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>lineEdit_pass</zorder>
   <zorder>pushButton_login</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_eror</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
